components/select/src/auro-select.js

```diff
@@ -545,11 +566,13 @@ export class AuroSelect extends LitElement {
           chevron
           part="dropdown">
           <span slot="trigger" aria-haspopup="true" id="triggerFocus">
-            ${this.value ? this.displayValue : html`<span class="placeholder">${this.placeholder}</span>`}
+            ${this.value ? this.displayValue : html`<span class="placeholder">${this.placeholder || this.labelText}</span>`}
           </span>
           <div class="menuWrapper">
           </div>
-          <slot name="label" slot="label"></slot>
+          <span slot="label" class="${!this.placeholder && !this.value ? 'hidden' : ''}">
```

Comment:
I am a little confused here. It looks like you are hiding the label when there is no placeholder and there is no value.

Shouldn't, like auro-input, we always show the elements label regardless of placehodler/value conditions?